docs/book/src/clusterctl/commands/move.md

```diff
@@ -52,7 +52,7 @@ while doing the move operation, and possible race conditions happening while the
 remediating etc. has never been investigated nor addressed.
 
 In order to avoid further confusion about this point, `clusterctl backup` and `clusterctl restore` commands have been
```

Comment:
I thought about if I should remove this whole section or just change deprecated to deleted. However I kept it as a reference. But I think it should definitely be deleted in the future.

Reply:
Nice catch, ok to leave this for now